apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -220,16 +219,14 @@ contract DisputableVoting is DisputableAragonApp, IForwarder {
         require(_canExecute(vote_), ERROR_CANNOT_EXECUTE);
 
         vote_.status = VoteStatus.Executed;
+        // TODO: check agreement action is not closed
```

Comment:
I wonder if we should detect this case from the Agreement—if the `msg.sender` is the action's disputable, we could ignore the `canClose()` check and immediately return if it's already been closed from the Agreement's side?

Reply:
It is an interesting approach. However, this comment was because the Agreement app must ensure actions are not closed multiple times (the locked collateral can be unlocked only once). So even if we skip the `canClose` check, `closeAction` should fail. 

Ideally, I was thinking to rename `_closeAgreementAction` to `_closeAgreementActionIfNecessary` in the base app or similar. I think you get the approach, simply to call `closeAction` if the action was not closed already.